app/connectors/optout/ITSAStatusUpdateConnector.scala

```diff
@@ -74,4 +75,36 @@ class ITSAStatusUpdateConnector @Inject()(val http: HttpClient, val appConfig: F
       }
     }
   }
+
+  /* todo this call to replace requestOptOutForTaxYear */
+  def makeITSAStatusUpdate(taxYear: TaxYear, taxableEntityId: String, updateReason: String)
```

Comment:
We are in process of migrating to HttpClientV2, please use new version of http client as this one is deprecated already